README.md

```diff
@@ -1856,6 +1856,7 @@ _Related: [Software Development - IDE & Tools](#software-development---ide--tool
 
 - [Appsmith](https://www.appsmith.com/) - Cloud or self-hosted open-source platform to build admin panels, CRUD apps and workflows. Build everything you need, 10x faster. ([Source Code](https://github.com/appsmithorg/appsmith)) `Apache-2.0` `Java/Docker`
 - [Budibase](https://www.budibase.com) - Build and automate internal tools, admin panels, dashboards, CRUD apps, and more, in minutes. Budibase is the open source alternative to Outsystems, Retool, Mendix, Appian. ([Source Code](https://github.com/Budibase/budibase)) `GPL-3.0` `Nodejs`
+- [Dify](https://dify.ai/) - Dify aims to enable developers (and even non-developers) to quickly build useful applications based on large language models, ensuring they are visual, operable, and improvable. ([Demo](https://udify.app/chat/F2Y4bKEWbuCb1FTC)), ([Source Code](https://github.com/langgenius/dify)) `MIT` `Python`
```

Comment:
```suggestion
- [Dify](https://dify.ai/) - Dify aims to enable developers (and even non-developers) to quickly build useful applications based on large language models, ensuring they are visual, operable, and improvable. ([Demo](https://udify.app/chat/F2Y4bKEWbuCb1FTC), [Source Code](https://github.com/langgenius/dify)) `MIT` `Docker`
```

Reply:
The only documented install method both in the GitHub Repo and on your Website only show Docker as a valid installation Methode.